github.tsv

```diff
@@ -83,3 +83,19 @@ https://github.com/OpenSDP/faketucky
 https://github.com/penzance/canvas_python_sdk
 https://github.com/penzance/harvard-data-tools
 https://github.com/refinery-platform/refinery-platform
+https://github.com/MCZbase/MCZbase
+https://github.com/MCZbase/schema_MCZbase
```

Comment:
I opened #19 because https://github.com/MCZbase/schema_MCZbase is a 404. My fault for not checking all of these before merging!